draft-scenarios/campaigns/castle_greek_gift.tex

```diff
@@ -0,0 +1,274 @@
+
+\textbf{Town Buildings:} \svgunit{bronze} Dwelling, Citadel
+
+\subsection*{\MakeUppercase{AI Hero setup}}
+
+\textbf{Faction:} Necropolis
+
+\textbf{Enemy Army:} A Pack of Skeletons, A Pack of Zombies, A Few Dread Knights, A Few Ghost Dragons
+
+\textbf{Enemy Deck:} 4 x Might cards
+
+\textbf{Special:} Place the Cloak of the Undead King IV and VI cards aside
+
+\subsection*{\MakeUppercase{Map setup}}
+
+Take the following Map Tiles and arrange them as shown in the Scenario map layout:
+
+\textbf{1 × Starting (I) Map Tile}
+\begin{itemize}
+    \item 1 x Castle (S3)
+\end{itemize}
+
+\textbf{3 × Far (II--III) Map Tile}
+\begin{itemize}
+    \item 3 x Castle (F3, F6, F9)
+\end{itemize}
+
+\textbf{2 × Near (IV--V) Map Tile}
+\begin{itemize}
+    \item 2 x Castle (N3, N6)
+\end{itemize}
+
+\subsection*{\MakeUppercase{Heroes placement}}
+
+Place your Castle Hero on the center Field of the S3 Castle starting map Tile.
+
+Place a Necropolis Hero on the center Field of the N3 Castle Tile.
+
+\subsection*{\MakeUppercase{Victory Conditions}}
+
+Defeat the enemy army.
+
+\subsection*{\MakeUppercase{Defeat Conditions}}
+\begin{itemize}
+  \item You lose one combat encounter
+  \item You run out of time at the end of the 8th Round
+\end{itemize}
+\end{multicols*}
+
+\newpage
+
+\begin{multicols}{2}
+
+\subsection*{\MakeUppercase{Timed Events}}
+
+\textbf{\nth{1} Round:}
+\begin{itemize}
+  \item Read ``Distant Threat'' section
+\end{itemize}
+
+\textbf{If entering the settlement on F3:}
+\begin{itemize}
+  \item Read ``Reverent Allies'' section
+\end{itemize}
+
+\textbf{Before combat with the enemy Hero:}
+\begin{itemize}
+  \item Read ``Bait and Switch'' section
+\end{itemize}
+
+\textbf{When you complete the Scenario:}
+\begin{itemize}
+  \item Read ``The Chase Begins'' section
+\end{itemize}
+
+\subsection*{\MakeUppercase{Additional rules}}
+
+During this Castle campaign Scenario, the following rules apply:
+
+\begin{itemize}
+  \item The enemy army does not move
+  \item Your Hero's maximum Level is 4
+  \item The Glory of Erathia building is unavailable
+  \item The Settlement on F3 does not have neutral enemies
+\end{itemize}
+
+\columnbreak
+
+\includegraphics[width=\linewidth, keepaspectratio]{\art/griffin.png}
+
+\end{multicols}
+
+\begin{tikzpicture}[overlay]
+  \node at (8.6, -5.3) {
+    \includegraphics[width=\textwidth]{\_assets/maps/castle_greek_gift.png}
+  };
+  \node at (10, -2.0) {\large{{\textbf{\textcolor{darkcandyapplered}{F3}}}}};
+\end{tikzpicture}
+
+\newpage
+
+\begin{multicols*}{2}
+
+\subsection*{\MakeUppercase{The story}}
+
+\textbf{Distant Threat}
+
+Catherine Ironfist rubbed a tired and worn thumb across the image of her father, King Gryphonheart, in a locket.
+She wanted to remember the good, valiant parts of his life rather than his actions in unlife.
+Her gaze turned to her cracked skin and mangled thumbnail, and it reminded her of how hard she had fought to be where she was.
+
+The mantle of Erathia was hers.
```

Comment:
```suggestion
The mantle of Erathia rested on her shoulders.
```
This a prominent example, I had to look up what "mantle of something" means 😆 IMO this is more self-explanatory.